### ISSUE TITLE
NRE in Message.get_Attachments after updating from MSGReader 4.4.4 to 4.4.5

### ISSUE THREAD

**Issue by dominik1996** [Sicos1977/MSGReader#306]
**Describe the bug**
Updating from MSGReader 4.4.4 to 4.4.5 causes a NullReferenceException reading MsgReader.Outlook.Storage.Message Attachments on specific files.

**To Reproduce**
```
using var message = new Storage.Message(MsgWithRtfBody);

var attachments = message.Attachments;
```

This code causes a NRE:

```
'message.Attachments' threw an exception of type 'System.NullReferenceException' at MsgReader.Outlook.Storage.Message.get_Attachments()
```

I cannot attach the specific file publicly, if the file is needed please let me know a way to send it to you.

**Comment by Cyril12740** [comment]
Yes i have the same issue and i had made an pull request to resolved the exceptions #307.

**Comment by Sicos1977** [comment]
I merged your pull request, i'll make a new nuget package tomorrow (already late overhere)

**Comment by Sicos1977** [comment]
Just released a new nuget package
